hermes-api/src/main/java/pl/allegro/tech/hermes/api/SubscriptionPolicy.java

```diff
@@ -13,44 +13,60 @@
     private static final int DEFAULT_RATE = 400;
     private static final int DEFAULT_MESSAGE_TTL = 3600;
     private static final int DEFAULT_MESSAGE_BACKOFF = 100;
+    private static final int DEFAULT_REQUEST_TIMEOUT = 1000;
+    private static final int DEFAULT_INFLIGHT_SIZE = 100;
```

Comment:
In practice CONSUMER_INFLIGHT_SIZE can be used as max value that user can specify. This property can be changed dynamically without having to modify subscription policy.